tests/integration/test_load_and_serve_agent.py

```diff
@@ -0,0 +1,185 @@
+import logging
+import os
+import subprocess
+from datetime import datetime
+from pathlib import Path
+from typing import Any
+
+import pytest
+from litellm.utils import validate_environment
+
+import any_agent.serving
+from any_agent import AgentConfig, AgentFramework, AnyAgent, TracingConfig
+from any_agent.config import MCPStdio, ServingConfig
+
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import (
+    InMemorySpanExporter,
+)
+from opentelemetry.sdk.trace import ReadableSpan
+
+if any_agent.serving.serving_available:
+    from common.client import A2AClient
+    from common.types import Message, TextPart
+
+FORMAT = "%(message)s"
+logging.basicConfig(
+    format=FORMAT,
+    datefmt="[%X]",
+)
+logger = logging.getLogger("any_agent_test")
+logger.setLevel(logging.DEBUG)
```

Comment:
hmm if this type of logging code is going to exist in multiple files, should a new "logging" function be created in the tests folder to reduce code repetition?